pkg/apps/apiserver/apiserver.go

```diff
@@ -132,7 +132,7 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 	v1Storage["deploymentConfigs/scale"] = deployConfigScaleStorage
 	v1Storage["deploymentConfigs/status"] = deployConfigStatusStorage
 	v1Storage["deploymentConfigs/rollback"] = deployConfigRollbackStorage
-	v1Storage["deploymentConfigs/log"] = deploylogregistry.NewREST(openshiftInternalAppsClient.Apps(), kubeInternalClient.Core(), kubeClient.CoreV1())
+	v1Storage["deploymentConfigs/log"] = deploylogregistry.NewREST(openshiftInternalAppsClient.Apps(), kubeClient.CoreV1(), kubeClient.CoreV1())
```

Comment:
why'd you give it the same client twice?

Reply:
NewREST has different requirements (pod client vs. replication controller client...) (we are doing this in other places too).

Reply:
@mfojtik I don't get this. It used to get internal and external kube client - why does it need to get the exact same external client twice now?

Reply:
@tnozicka different interfaces on input args... i can rework the REST, but except being ugly here I don't see that much gain.

Reply:
i converted it to kubernetes.Interface{}

Reply:
> @tnozicka different interfaces on input args... i can rework the REST, but except being ugly here I don't see that much gain.

this is the only caller of that method. I agree this is cosmetic change which is fine as a follow up